pkg/blockio/blockio_default.go

```diff
@@ -0,0 +1,39 @@
+//go:build !linux
```

Comment:
nit: `blockio_default.go` might be a bit misleading here?
Possibly `blockio_others.go` or `blockio_nonlinux.go` or `blockio_unsupported.go`?

Reply:
Thanks! Updated